### PR TITLE
Update labels w/ traffic data and w/o some exchange data

### PR DESCRIPTION
**Pull request by BoltzmannBrain**
@subutai these labels include the traffic data, and exclude the eliminated exchange data.

### CODE REVIEW

File: labels/combined_labels.json [hunk 1]
```diff
@@ -159,27 +135,55 @@
         "2014-07-17 09:50:00"
     ],
     "realKnownCause/mole_key_updown.csv": [
```
Comment (subutai) [inline labels/combined_labels.json]: Shouldn't use the term "mole". Could be rogue_agent_updown

File: labels/combined_labels.json [hunk 2]
```diff
@@ -227,5 +231,6 @@
     "realTweets/Twitter_volume_UPS.csv": [
         "2015-03-24 18:17:53",
         "2015-03-29 16:27:53"
-    ]
+    ],
+    "tmp/speed_7578.csv": []
```
Comment (subutai) [inline labels/combined_labels.json]: What is this file?

File: labels/combined_windows.json [hunk 3]
```diff
@@ -378,11 +318,15 @@
     ],
     "realKnownCause/mole_key_updown.csv": [
```
Comment (subutai) [inline labels/combined_windows.json]: Shouldn't use mole.

File: labels/combined_windows.json [hunk 4]
```diff
@@ -567,5 +581,6 @@
             "2015-03-28 07:27:53.000000",
             "2015-03-31 01:27:53.000000"
         ]
-    ]
+    ],
+    "tmp/speed_7578.csv": []
```
Comment (subutai) [inline labels/combined_windows.json]: ??

**Comment by BoltzmannBrain** [comment]
Woah sorry clearly some stash issues, I'll clean it up

**Comment by BoltzmannBrain** [comment]
@subutai should be all set now.

**Comment by subutai** [comment]
:+1:

**Pull request merged**